pkg/scheduler/internal/parallelize/parallelism.go

```diff
@@ -23,21 +23,24 @@ import (
 	"k8s.io/client-go/util/workqueue"
 )
 
-const parallelism = 16
+// Parallelism is used the store value from scheduler config
+var Parallelism = 16
```

Comment:
I expected this to go away now that there was a config option

Reply:
I don't this there is a simpler way to do this as we have different invocations of the scheduler (some load defaults and some don't), so we need a way to set parallelism at the package level. @alculquicondor What do you recommend ?

Reply:
Right, some plugins' tests run without a scheduler object, so we would have to configure it (or pass as parameter) for every test and every external user (autoscaler, descheduler, etc).

This seems sufficient to me for now, but we can revisit in the future.

Reply:
Global mutable package vars are still fragile, so I'm not thrilled about plumbing this this way. At the very least we should document the expectations (should only be set once, should be set from the main goroutine before starting any background goroutines that will read it). Ideally this would be protected via a setter that only lets the parallelism be set once on normal scheduler invocations and ensures the value is > 0

Reply:
A setter `MustSetParallelism` sgtm. This should panic if called more than once or value is <= 0